src/TestScope.js

```diff
@@ -77,35 +77,40 @@ export default class TestScope {
   //
   // label - Label for these test cases.
   // f     - Callback function containing your tests cases defined with `it`.
+  // tag   - (Optional) A string tag used to determine whether the group of
+  //          tests should run. Defaults to null.
   //
   // Example
   //
   //   // specs/MyFeatureSpec.js
   //   export default function(spec) {
   //     spec.describe('My Scene', function() {
-  //
   //       spec.it('Has a component', async function() {
   //         await spec.exists('MyScene.myComponent');
-  //       });
-  //
+  //       }, 'focus');
```

Comment:
How do we feel about this being at the end of the argument list? Is this for easy backwards compat?

Reply:
Do you mean you don't like the way it looks? When I was reviewing, I actually tried changing it to be the second argument:

```js
export default function(spec) {
  spec.describe('A test', 'focus', function() {
    spec.it('works', async function() {
      await spec.exists('Something');
    });
  });
}
```

... but in my opinion this makes it a lot less obvious which tests are going to run when you're scanning down a file.

It _also_ makes backwards compatibility a lot easier.

Reply:
I can't think of a better way to do it, so yeah let's stick like this.